programs/rodeos/CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
-add_executable( ${RODEOS_EXECUTABLE_NAME} 
+add_executable( ${RODEOS_EXECUTABLE_NAME}
   cloner_plugin.cpp
+  streamer_plugin.cpp
```

Comment:
I think this would be better as:

```
file(GLOB SRC *.cpp *.hpp streams/*.hpp)
add_executable( ${RODEOS_EXECUTABLE_NAME}
  ${SRC}
)
```

Reply:
If you do that, you really ought to add `CONFIGURE_DEPENDS`. Even then it's not recommended by cmake

> We do not recommend using GLOB to collect a list of source files from your source tree. If no CMakeLists.txt file changes when a source is added or removed then the generated build system cannot know when to ask CMake to regenerate. The CONFIGURE_DEPENDS flag may not work reliably on all generators, or if a new generator is added in the future that cannot support it, projects using it will be stuck. Even if CONFIGURE_DEPENDS works reliably, there is still a cost to perform the check on every rebuild.

Reply:
We glob header files in other CMakeLists.txt and add that in. Is that the preferred method?

Reply:
>CONFIGURE_DEPENDS

Actually, this is a cmake 3.12 feature